lib/feed-bird.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function(bird, food, cb) {
+  console.log(`Feeding ${bird} to the ${food}...`);
+  setTimeout(cb, 500);
```

Comment:
Lets introduce promises in the refactor: https://github.com/yola/farm-foo/issues/2